index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  
+ <header id="header">
+  <nav id="nav-bar" class="navbar navbar-light bg-light">
+    <div class="container">
+      <a class="navbar-brand" href="#">
+        <img id="header-img" src="img/microverse.png" alt="" width="30" height="24">
+      </a>
+      
+          <a class="nav-link" aria-current="page" href="#features">Features</a>
+          <a class="nav-link" href="#pricing">Pricing</a>
+          <a class="nav-link" href="#contactus">Contact Us</a>
+    </div>
+  </nav>
+</header>
+
+<div>
+  <video id="video" controls="controls">
+    <source src="img/Video_Tour_Workspace.mp4" type="video/mp4">
+    Your browser does not support the HTML5 Video element.
+  </video>
+</div>
+<form id="form">
+  <div class="mb-3">
+    <input type="email" class="form-control" id="email" aria-describedby="emailHelp" placeholder="Enter your e-mail">
+   
+  </div>
+  
+  <button id="submit" href=" https://www.freecodecamp.com/email-submit"  type="submit" class="btn btn-primary">Submit</button>
+</form>
+
+
+  <h1>HELLO WORLD, we are software developers</h1>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+
+
+  <!--
+
+      h
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
Kindly, do not leave commented and unused code on your codebase. This makes your codebase unclean and hard to read.